### PR TITLE
termius: init at 5.10.1

### LINKED ISSUE #74777: Package request: termius (deb, Electron app)
**Project description**
> Termius is the SSH client that works on Desktop and Mobile.

A cross-platform SSH client with cloud data sync and more. 

**Metadata**

* homepage URL: https://termius.com/
* source URL: https://www.termius.com/download/linux/Termius.deb
* license: unfree
* platforms: linux

### LINKED ISSUE #76816: Termius ssh client
**Project description**
I'm using Termius everyday because it is quite simple to easy and it allows me to share my ssh keys between my different computers and even my phone. I really like this app and it's possible to get the premium features for free with github student. Currently I have to restart to windows in order to use it and I would really like to be able to use it on nixos too :)

**Metadata**

* homepage URL: https://termius.com/
* source URL: this is closed source
* license: I'm not sure but I can ask them
* platforms: this is an electron app so there is a version for basically every systems

### PR DESCRIPTION
**Pull request by Br1ght0ne**
###### Motivation for this change
Closes #74777 and closes #76816.
Mostly copied from https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/instant-messengers/wire-desktop/default.nix.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
```
/nix/store/04p269sdm2hzzgsld4kdrc1x6jw6az3p-termius-5.2.3	 527.7M
```
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

cc @Th0rgal

---
This change is [Reviewable](https://reviewable.io/reviews/nixos/nixpkgs/76891)

### CODE REVIEW

File: pkgs/applications/networking/termius/default.nix [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "termius";
+  version = "5.2.3";
+
+  src = fetchurl {
+    # Sadly, versioned URLs are not a thing here.
+    url = "https://www.termius.com/download/linux/Termius.deb";
```
Comment (misuzu) [inline pkgs/applications/networking/termius/default.nix]: Proper URL: ```https://deb.termius.com/pool/main/t/termius-app/termius-app_5.2.3_amd64.deb```
For reference: ```wget -qO- https://deb.termius.com/dists/squeeze/main/binary-amd64/Packages```

**Review by misuzu — CHANGES_REQUESTED** [review]

**Comment by filalex77** [comment]
Thanks @misuzu!

**Review by misuzu — APPROVED** [review]

File: pkgs/applications/networking/termius/default.nix [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+, udev
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "termius";
+  version = "5.3.2";
+
+  src = fetchurl {
+    url = "https://deb.termius.com/pool/main/t/termius-app/termius-app_5.3.2_amd64.deb";
```
Comment (nh2) [inline pkgs/applications/networking/termius/default.nix]:
```nix
"termius-app_5.3.2_amd64.deb"
```

->

```nix
"termius-app_${version}_amd64.deb"
```

**Comment by Th0rgal** [comment]
Hey @fifalex77 could you update your PR? Here is what I did: https://github.com/Th0rgal/nixpkgs/blob/35c91b27e19e82018eeea3232d572b0d87ad816a/pkgs/applications/networking/termius/default.nix

**Comment by filalex77** [comment]
@GrahamcOfBorg build termius

**Pull request merged**